docs/deployment-instructions.md

```diff
@@ -10,38 +10,71 @@ This below diagram depicts the high-level overview of the eHSM-KMS in K8S cluste
 ---
 
 ## Prerequisites
-* Ensure you already have a running kubenetes cluster environment, if not, please follow [k8s-setup-guide](k8s-setup-guide.md) to setup the K8S cluster.
-* Ensure you already have a NFS server, if not, please follow [nfs-setup-guide](nfs-setup-guide.md) to setup a nfs server.
+* A running kubenetes cluster environment, if not, please follow [k8s-setup-guide](k8s-setup-guide.md) to setup the K8S cluster.
+    * The worker nodes MUST support SGX(need FLC support for DCAP remote attestation).
+    * The Master is a generic node.
+* A centralized dkey-provisioning server node.
+    * MUST support SGX (w/ FLC) OR directly connect with a dedicated HSM.
```

Comment:
Do we need to mention dedicated HSM?

Reply:
our reference solution is target for enclave,  but better to mention it as another choice.